test/node.js

```diff
@@ -266,13 +266,13 @@ node.expectedFeeForTrsWithData = function (amount) {
 
 // Returns a random username of 16 characters
 node.randomUsername = function () {
-	var randomLetter = randomString.generate({
+	var randomLetter = node.randomString.generate({
```

Comment:
why first char need to be alphabetic lowercase?

Reply:
It's just a choice.